common/persistence/visibility/store/elasticsearch/query_interceptors_test.go

```diff
@@ -148,12 +148,12 @@ func (s *QueryInterceptorSuite) TestDurationProcessFunc() {
 		value     interface{}
 		returnErr bool
 	}{
-		{value: "1", returnErr: false},
+		{value: nil, returnErr: true},
 		{value: 1, returnErr: false},
 		{value: int64(18180000000000), returnErr: false},
 		{value: int64(1000000000), returnErr: false},
 		{value: nil, returnErr: true},
-		{value: "bad value", returnErr: false},
+		{value: nil, returnErr: true},
```

Comment:
same case 3 times?

Reply:
and why did you remove "bad value" case?

Reply:
These are not the cases, these are the expected values for the test cases above.